src/main/java/org/usfirst/frc4048/common/gyro/RealGyroIO.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc4048.common.gyro;
+
+import com.kauailabs.navx.frc.AHRS;
+
+public class RealGyroIO implements GyroIO {
+    private final ThreadedGyro gyro;
+    private double angleOffset = 0;
+
+    public RealGyroIO() {
+        this.gyro = new ThreadedGyro(new AHRS());
+        gyro.start();
+    }
+
+    @Override
+    public void setAngleOffset(double offset) {
+        this.angleOffset = offset;
```

Comment:
Why do we need this here? You can cache this in the `ThreadedGyro` if you want to save hardware access

Reply:
done